lib/evss/aws/reference_data/configuration.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module EVSS
+  module AWS
+    module ReferenceData
+      class Configuration < EVSS::AWS::Configuration
+        def base_path
+          # TODO: integrate with Settings.yml & devops
+          'https://internal-staging-services-1341723990.us-gov-west-1.elb.amazonaws.com/api/refdata/v1'
```

Comment:
Could wiring up settings.yml be part of this PR? I believe any devops changes would just override that (no need to coordinate PRs going out)

Reply:
Agreed, and I don't really want this endpoint merged into vets-api repo. It's not particularly sensitive since it has plenty of other access control around it, but better safe than sorry.